test/binder/binder_test.cpp

```diff
@@ -36,7 +36,7 @@ using std::make_tuple;
 namespace peloton {
 namespace test {
 
-class BinderCorrectnessTest : public PelotonTest {
+class BinderCorrectnessTests : public PelotonTest {
```

Comment:
@tcm-marcel At this stage, there are parent classes which still have the `Test` suffix instead of the `Tests` suffix. They are defined in `.h` files, and thus were not changed by the name-fixing script I ran. If I change them, I would also have to change their name in inheritance statements, e.g., here the statement will look like `class BinderCorrectnessTests : public PelotonTests {`. Should I do that?

Reply:
@schedutron Yes.